website1/css/sections/first-page.css

```diff
@@ -0,0 +1,31 @@
+.first-page {
+  position: relative;
+  background: url("../../img/section1_img1.png") center / cover no-repeat;
+  padding-top: 214px;
+  padding-bottom: 130px;
+}
+
+.first-page__content {
+  display: flex;
+  flex-direction: column;
+  color: white;
+}
+
+.first-page__text_narrow {
+  font-size: 32px;
+  line-height: 127%;
+}
+
+.first-page__text_wide {
+  margin-top: 5px;
+  font-weight: 600;
+  font-size: 86px;
+  line-height: 127%;
+}
+
+.first-page__text_default20 {
```

Comment:
In the future write the name of the classes more clearly for example `title`, `subtitle`, or create a block with the name ~ `text` with different modifications.